src/style.css

```diff
@@ -0,0 +1,51 @@
+@font-face {
+  font-family: Avenir;
+  src: url("fonts/Avenir.ttc");
+}
+
+.container {
+  margin-top: 20px;
+}
+
+.searchForm {
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+}
+
+.search {
+  width: 100%;
+  outline: none;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  color: #3d4e61;
+  box-sizing: border-box;
+  font-family: Avenir, sans-serif;
+  background-image: url("images/Search.svg");
+  background-repeat: no-repeat;
+}
+
+.search:hover {
+  border: 1px solid aqua;
```

Comment:
aqua? You must take design from the [mockup](https://www.figma.com/file/kf3AWulK9elrNk34wtpjPw/Airbnb-Search-bar?node-id=0%3A1)